src/main/java/org/jabref/model/study/Study.java

```diff
@@ -0,0 +1,119 @@
+package org.jabref.model.study;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.UnknownField;
+
+/**
+ * This class represents a scientific study.
+ *
+ * This class defines all aspects of a scientific study relevant to the application. It is a proxy for the file based study definition.
+ */
+public class Study {
+    private static final String STUDY_ENTRY_NAME = "study";
```

Comment:
Please introocude a new class "SystematicLiteratureReviewStudyEntryType" (or similar) in the package `org.jabref.model.entry.types` and rewort the code accordingly.